### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by thisisjvgrace** [aikaterna/gobcog#93]
This happens once in a while and after this error occurs, bot keeps saying "There's already another adventure going on in this server." and new adventure only happens if I manually reload the cog.

```py
Feb 04 01:55:40 ec2 redbot[22606]: [2020-02-04 01:55:40] [ERROR] red.cogs.adventure: Something went wrong controlling the game
Feb 04 01:55:40 ec2 redbot[22606]: Traceback (most recent call last):
Feb 04 01:55:40 ec2 redbot[22606]:   File "/home/ubuntu/v3/v3/cogs/CogManager/cogs/adventure/adventure.py", line 3408, in _adventure
Feb 04 01:55:40 ec2 redbot[22606]:     reward, participants = await self._simple(ctx, adventure_msg, challenge)
Feb 04 01:55:40 ec2 redbot[22606]:   File "/home/ubuntu/v3/v3/cogs/CogManager/cogs/adventure/adventure.py", line 3531, in _simple
Feb 04 01:55:40 ec2 redbot[22606]:     await self._choice(ctx, adventure_msg)
Feb 04 01:55:40 ec2 redbot[22606]:   File "/home/ubuntu/v3/v3/cogs/CogManager/cogs/adventure/adventure.py", line 3649, in _choice
Feb 04 01:55:40 ec2 redbot[22606]:     return await self._result(ctx, adventure_msg)
Feb 04 01:55:40 ec2 redbot[22606]:   File "/home/ubuntu/v3/v3/cogs/CogManager/cogs/adventure/adventure.py", line 4147, in _result
Feb 04 01:55:40 ec2 redbot[22606]:     text += await self._reward(
Feb 04 01:55:40 ec2 redbot[22606]:   File "/home/ubuntu/v3/v3/cogs/CogManager/cogs/adventure/adventure.py", line 5397, in _reward
Feb 04 01:55:40 ec2 redbot[22606]:     chest_type = types[special.index(1)]
Feb 04 01:55:40 ec2 redbot[22606]: IndexError: list index out of range
```

**Comment by skylarr1227** [comment]
I had this issue, it was from the JSON having data from previous adventure before the update.

Unload Cog
Shutdown bot
Delete the settings.json in the cogs folder
Start the bot
Load the cog
It will regen the settings.json file

Should fix the issue your having.

**Comment by Drapersniper** [comment]
Think I've fixed it ... do you happen to remeber if this happened during a boss fight? @thisisjvgrace

**Comment by aikaterna** [comment]
Theoretically resolved with 1611c0c, please leave an issue with repro steps if you run into this again.
